shared/menubar/index.desktop.js

```diff
@@ -117,22 +116,21 @@ class MenubarRender extends React.Component<Props, State> {
             color={Styles.globalColors.yellow}
           />
           <Kb.Text type="Body" style={{alignSelf: 'center', marginTop: 6}}>
-            You're logged out of Keybase!
+            You are logged out of Keybase.
           </Kb.Text>
           <Kb.ButtonBar direction="row">
-            <Kb.Button label="Log In" onClick={this.props.logIn} />
+            <Kb.Button label="Log in" onClick={this.props.logIn} />
           </Kb.ButtonBar>
         </Kb.Box>
       </Kb.Box>
     )
   }
 
   _renderDaemonHandshakeWait() {
-    const menuColor = this.state.showingMenu ? Styles.globalColors.black_50 : Styles.globalColors.black_50
     const text =
       this.props.daemonHandshakeState === 'waitingForWaiters'
-        ? `Connecting UI services to crypto engine... This may take a few seconds`
-        : `Starting up Keybase`
+        ? `Connecting interface to crypto engine... This may take a few seconds.`
```

Comment:
should this be "the interface"?

Reply:
I think UI language can be article-free in certain cases: "Loading dependencies...", "Syncing folders...", etc.